hera_qm/xrfi.py

```diff
@@ -1398,10 +1413,70 @@ def xrfi_run(ocalfits_file=None, acalfits_file=None, model_file=None, data_file=
         The omnical calfits file to use to flag on gains and chisquared values.
     acalfits_file : str, optional
         The abscal calfits file to use to flag on gains and chisquared values.
+    medfilt : bool, optional
+        Determine whether to run a round of median filtering
     model_file : str, optional
         THe model visibility file to flag on.
     data_file : str, optional
         The raw visibility data file to flag.
+    omnical_median_filter : bool, optional
+        If true, run a median filter on omnical gains.
```

Comment:
For all of these, I'd clarify that if the source file(s) is None, they aren't run even if True.

Reply:
I've added this clarification.